### PR TITLE
add pdfgrep 2.1.2 (linux and macos only)

### PR DESCRIPTION
**Pull request by julozi**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by julozi** [comment]
pdfgrep does not support Windows build so CI errors look alright.

**Comment by julozi** [comment]
Hello @conda-forge/staged-recipes
pdfgrep recipe is ready for review

File: .ci_support/azure-linux-64-comp7.yaml [hunk 1]
```diff
@@ -8,3 +8,29 @@ channel_sources:
   - conda-forge,defaults
 docker_image:
   - condaforge/linux-anvil-comp7
```
Comment (xhochy) [inline .ci_support/azure-linux-64-comp7.yaml]: Can you remove this spurious edit?

Comment (julozi) [reply .ci_support/azure-linux-64-comp7.yaml]: Oops I restored original version. Sorry for that.

**Pull request merged**